debian/changelog

```diff
@@ -1,3 +1,11 @@
+opx-nas-linux (5.25.0+opx1) unstable; urgency=medium
+
+  * Bugfix: Mac for L3 ARP fail to relearn after script flapping the
```

Comment:
Suggestion: L3 ARP MAC relearning fails after the port-channel is flapped and switch is reloaded.